src/main/java/com/openshift/internal/restclient/capability/resources/AbstractOpenShiftBinaryCapability.java

```diff
@@ -152,9 +158,22 @@ public final void start(final OpenShiftBinaryOption... options) {
 	}
 	
 	private void startProcess(final String location, final OpenShiftBinaryOption... options) {
-		String cmdLine = new StringBuilder(location).append(' ').append(buildArgs(Arrays.asList(options))).toString();
-		String[] args = StringUtils.split(cmdLine, " ");
-		ProcessBuilder builder = new ProcessBuilder(args);
+		List<String> args = new ArrayList<String>();
+		ProcessBuilder builder = null;
```

Comment:
i believe, that for improved readability, we could have l161-l177 extracted to their own method 
```
ProcessBuilder createProcessBuilder(String location, OpenShiftBinaryOption... options)
```